projects/storefrontapp-e2e-cypress/cypress/e2e/accessibility/product-details-page.a11y-e2e.cy.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * SPDX-FileCopyrightText: 2025 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as siteContextSelector from '../../helpers/site-context-selector';
+
+/**
+ * This test checks accessibility concerns on the PDP page using Access Continuum
+ */
+describe('Product Details Page Accessibility', () => {
+  beforeEach(() => {
+    cy.a11yContinuumSetup();
+    cy.visit(siteContextSelector.PRODUCT_PATH_2);
+  });
+
+  context('PDP main section', () => {
+    it('should pass a11y check', () => {
```

Comment:
```suggestion
    it('PDP main section', () => {
```
I think we don't need separate context for each test.